resources/commands.json

```diff
@@ -983,6 +983,31 @@
 						"description": "The NFT type"
 					}
 				]
+			}, {
+				"name": "balance [nftAddress]",
+				"description": "It gets the balance of a NFT",
+				"details": "This command allows to check if an address is a holder of a specific NFT wrapped by a Nevermined DID",
+				"examples": ["ncli nfts721 hold did:nv:afd733c23c41af948be7ec039c3fb2048d437e082a69ea3f336cdf452a49be7e"],
+				"commandHandler": "balanceNft",
+				"positionalArguments": [{
+					"name": "nftAddress",
+					"type": "string",
+					"description": "The NFT contract address"
+				}],
+				"optionalArguments": [{
+						"name": "address",
+						"type": "string",
+						"default": "",
+						"description": "The address of the account to get the balance"
```

Comment:
should this be an optional argument?

Reply:
I think so because if is not given it try  to get it from the DDO